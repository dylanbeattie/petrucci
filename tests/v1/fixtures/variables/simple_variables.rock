Variable is 1
Say variable

Value is 1
say value with value

X is 1
Y is 2
Say x plus y
Alpha says r
Beta says o
Gamma says c
Delta says k
Say alpha with beta with gamma with delta

Let themes be 10 (variable name beginning with pronoun 'the')
Let italics be true (variable name beginning with pronoun 'it')
Shout themes
Shout italics
