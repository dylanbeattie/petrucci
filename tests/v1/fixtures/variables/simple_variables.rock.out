1
2
3
rock
10
true
